tracer/src/Datadog.Tracer.Native/iast/method_info.cpp

```diff
@@ -206,6 +206,10 @@ namespace iast
     {
         return _id;
     }
+    SignatureInfo* MethodSpec::GetMethodSpecSignature()
+    {
+        return MemberRefInfo::GetSignature();
```

Comment:
That really looks like a static method call. God, C++ is so confusing

Reply:
Yeah, multiple inheritance shenanigans...